qubes.SaltLinuxVM

```diff
@@ -29,7 +29,7 @@ read target_vm
 read salt_command
 
 rm -rf /srv
-mv /home/user/QubesIncoming/dom0/srv /srv
+mv ~/QubesIncoming/dom0/srv /srv
```

Comment:
This does break the service. `qvm-copy-to-vm` called from dom0 copies the files into default user home, while this service is called as root. This results in:
```
[2023-04-26 05:45:01] [   33.310217] qubes.SaltLinuxVM-dom0[615]: mv: cannot stat '/root/QubesIncoming/dom0/srv': No such file or directory
```
I guess you need to get default-user from qubesdb here.

Reply:
What about copying the files from dom0 as root?  `qvm-copy-to-vm` is not used here.

Reply:
It is not copied as root, that's the point. It's copied as default user, and the files land in `/home/user/QubesIncoming/dom0` (or other for different user name). But this service is running as root, so `~/QubesIncoming` is the wrong one. The thing that copies them is python part running in dom0 - see the `qubessalt` dir.

Reply:
> What about changing the Python code to copy the files as root?

That would break compatibility with older templates (templates from older qubes version should function properly under newer qubes version, _especially_ in parts related to upgrade).